bin/node-template/pallets/template/src/lib.rs

```diff
@@ -57,7 +57,14 @@ pub mod pallet {
 	}
 
 	#[pallet::hooks]
-	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {}
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_finalize(block_number: T::BlockNumber) {
+			debug::info!("on finalize");
+		}
+		fn on_idle(block_number: T::BlockNumber) {
+			debug::info!("on idle");
+		}
+	}
```

Comment:
btw. this should probably go in the node, not the node template
node: demonstration
node-template: template for new projects

Reply:
ya, for sure, just testing it out for myself for now